lib/ansible/playbook/task.py

```diff
@@ -116,8 +117,24 @@ def get_path(self):
             path = "%s:%s" % (self._parent._play._ds._data_source, self._parent._play._ds._line_number)
         return path
 
+    def get_resolved_name(self):
```

Comment:
Isn't this feature intended for callbacks and module_defaults? Since I added tests for callbacks, I thought there should be a nicer interface for it - I assumed callbacks would mostly use the feature for task banners, but I guess I don't know what was planned.

Reply:
The original ask was actually just for Tower and analytics, so they'd know definitively which module actually ran from which collection instead of trying to reverse-engineer it from the task action. The feedback on having FQCNs show up in task banners in some of the collections prototypes was overwhelmingly negative, so I'd be surprised if that ever went anywhere ;)

Reply:
Okay, cool. I didn't know any of that, so I'll remove this. Thanks for the info. :-)